### DISCUSSION TITLE
Danski Jóhann er lagður í einelti - 2023/8/2

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#40] [vefsida-ummæli]
# Danski Jóhann er lagður í einelti - 2023/8/2

Danski Trausti leggur danska Jóhann í einelti

https://www.biskupsreglan.com/verkefni/danski-johann-einelti

**Comment by MatthiasGudni** [comment]
Hvað er í gangi

**Reply by TJ478** [reply]
Þetta er nú bara Danmörk
